DigitalLearningSolutions.Web/Controllers/LearningPortalController/LearningPortalController.cs

```diff
@@ -41,119 +37,18 @@ public partial class LearningPortalController : Controller
             this.filteredApiHelperService = filteredApiHelperService;
         }
 
-        public IActionResult AccessibilityHelp()
-        {
-            var accessibilityText = configService.GetConfigValue(ConfigService.AccessibilityHelpText);
-            if (accessibilityText == null)
-            {
-                logger.LogError("Accessibility text from Config table is null");
-                return StatusCode(500);
-            }
-
-            var model = new AccessibilityHelpViewModel(accessibilityText);
-            return View(model);
-        }
-
-        public IActionResult Terms()
-        {
-            var termsText = configService.GetConfigValue(ConfigService.TermsText);
-            if (termsText == null)
-            {
-                logger.LogError("Terms text from Config table is null");
-                return StatusCode(500);
-            }
-
-            var model = new TermsViewModel(termsText);
-            return View(model);
-        }
-
-        public IActionResult Error()
-        {
-            var model = GetErrorModel();
-            Response.StatusCode = 500;
-            return View("Error/UnknownError", model);
-        }
-
-        [Route("/LearningPortal/StatusCode/{code:int}")]
-        [IgnoreAntiforgeryToken]
-        public new IActionResult StatusCode(int code)
-        {
-            var model = GetErrorModel();
-            Response.StatusCode = code;
-
-            return code switch
-            {
-                404 => View("Error/PageNotFound", model),
-                403 => View("Error/Forbidden", model),
-                _ => View("Error/UnknownError", model)
-            };
-        }
-
-        private int GetCandidateId()
-        {
-            return User.GetCustomClaimAsRequiredInt(CustomClaimTypes.LearnCandidateId);
-        }
-
-        private int? GetCentreId()
-        {
-            return User.GetCustomClaimAsInt(CustomClaimTypes.UserCentreId);
-        }
         private string GetCandidateNumber()
         {
-            string? sCandNum = User.GetCustomClaim(CustomClaimTypes.LearnCandidateNumber);
-            if (sCandNum == null)
-            {
-                return "";
-            }
-            else
-            {
-                return sCandNum;
-            }
-        }
-        private string GetCandidateForename()

```

Comment:
This method and the one below (GetCandidateSurname) were unused so I've removed them. Let me know if they were there for a reason 🙂

Reply:
None that I can remember - they may need adding back in at some point but best wait until they are needed :)